code/__DEFINES/is_helpers.dm

```diff
@@ -22,7 +22,7 @@
 
 #define iseffect(A) (istype(A, /obj/effect))
 
-#define is_cleanable(A) (istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune)) //if something is cleanable
+#define is_cleanable(A) (istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune) && !istype(A, /obj/effect/rune/narsie)) //if something is cleanable
```

Comment:
A better way of doing this maybe would be to remake this into a proc rather than a define. That way it could just be overridden for anything you don't want cleaned, rather than having to add exceptions to the define every time. E.g:
```dm
/obj/proc/is_cleanable()
	return FALSE

...

/obj/effect/rune/is_cleanable()
	return TRUE

...

/obj/effect/rune/narsie/is_cleanable()
	return FALSE
```
Etc.

Reply:
Done, just forgot to mention earlier